allensdk/test/brain_observatory/behavior/test_behavior_lims_api.py

```diff
@@ -273,9 +273,8 @@ def test_stim_file_regression(self):
                 == self.od.extractor.get_behavior_stimulus_file())
 
     def test_get_rewards_regression(self):
-        """Index is timestamps here, so remove it before comparing."""
-        bd_rewards = self.bd.get_rewards().reset_index(drop=True)
-        od_rewards = self.od.get_rewards().reset_index(drop=True)
+        bd_rewards = self.bd.get_rewards().drop(columns=['timestamps'])
+        od_rewards = self.od.get_rewards().drop(columns=['timestamps'])
```

Comment:
Does this test fail if `timestamps` are present? If not, maybe makes sense to just keep them?

Reply:
yes it does fail.
https://github.com/AllenInstitute/AllenSDK/blob/fcf70e36cdeaeee79cc7a2b973d47069f47e5e54/allensdk/test/brain_observatory/behavior/test_behavior_lims_api.py#L252-L255

Reply:
ah right because one is from a behavior session (timing only from pkl) and the other is from behavior ophys session (timing aligned to sync)